python/lsst/meas/astrom/matchOptimisticB.py

```diff
@@ -3,14 +3,19 @@
 from builtins import object
 import math
 
+import numpy as np
+from scipy.spatial import cKDTree
+
+import lsst.afw.table as afwTable
 import lsst.pex.config as pexConfig
 import lsst.pipe.base as pipeBase
 from lsst.meas.algorithms.sourceSelector import sourceSelectorRegistry
 
 from .setMatchDistance import setMatchDistance
-from .astromLib import matchOptimisticB, MatchOptimisticBControl
+from .optimistic_pattern_matcher_b_3D import OptimisticPatternMatcherB
 
 __all__ = ["MatchOptimisticBTask", "MatchOptimisticBConfig"]
+__deg_to_rad__ = np.pi/180.
```

Comment:
I'd like to see the use `np.radians` instead.  It's just one less special global running around.